src/App.jsx

```diff
@@ -1,35 +1,11 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { Provider } from 'react-redux';
+import store from './redux/store';
+import Greeting from '../src/components/Greeting';
 
-function App() {
-  const [count, setCount] = useState(0)
+const App = () => (
+  <Provider store={store}>
+    <Greeting />
+  </Provider>
```

Comment:
Nice job so far😉. Your project looks great👍🏼. Just one minor issue😉. As a r[equirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md), the greeting component should be linked to the app component with the help of react-router, please do well to add this feature 😉.

<img width="881" alt="image" src="https://github.com/Kingno3l/Hello-React-Front-End/assets/64643995/17f96b32-65a3-46c1-bee7-e087cf92a4d7">